Code/Tools/TestImpactFramework/Frontend/Console/Common/Code/Source/TestImpactConsoleTestSequenceEventHandler.cpp

```diff
@@ -188,6 +199,16 @@ namespace TestImpact
             const auto progress =
                 AZStd::string::format("(%03zu/%03zu)", numTestRunsCompleted, totalNumTestRuns);
 
+            if (not testRun.GetStdOutput().empty())
+            {
+                std::cout << testRun.GetStdOutput().c_str();
+            }
+
+            if (not testRun.GetStdError().empty())
```

Comment:
nit: I know this is fine and allowed in C++ but I'd stick with `!` for consistency with the rest of the code base

Reply:
Fixed 👍

Reply:
This actually comes form C and is a historical thing because certain keyboards didn't have the operators we routinely use today - https://stackoverflow.com/a/2376456 😅